### DISCUSSION TITLE
Support for changing the UI fontsize

### DISCUSSION THREAD

**Discussion by shadeMe** [luminescence-software/Metatogger-Support#4] [Ideas]
The application appears to be DPI-aware but in fractional HiDPI contexts (my current setup is 5120x2160 @ 125% scaling), the UI's font is (subjectively) a bit on the smaller side. Having a way to increase the font size would be a great QoL feature.

**Comment by CyberSinh** [comment]
In version 7.7.1 of Metatogger, released today, the font size of the grid has been increased by one size (from 11 to 12).
Does this size suit you better? Does it meet your needs?

**Reply by shadeMe** [reply]
Ah, I haven't tried it yet - is there a full change log for 7.7, incidentally?

**Reply by CyberSinh** [reply]
You can read the change log [here](https://www.luminescence-software.org/en/metatogger/news/#release-of-metatogger-771).

**Reply by shadeMe** [reply]
I did a quick test and the font size is definitely readable atm, but if it's trivial to implement, I'd still appreciate a configurable setting for the font as a whole (just for the sake of accessibility; my vision is definitely not what it used to be).

**Reply by CyberSinh** [reply]
I'm leaving the suggestion open to allow other users to vote. I'll see how to implement this depending on the popularity of the request.
